[2023-08-05T10:40:28.357+0200] {processor.py:157} INFO - Started process (PID=82616) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:40:28.362+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T10:40:28.363+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:40:28.363+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:40:28.419+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:40:28.419+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T10:40:28.420+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:40:28.419+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T10:40:28.420+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:40:29.854+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.499 seconds
[2023-08-05T10:44:05.585+0200] {processor.py:157} INFO - Started process (PID=82700) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:44:05.597+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T10:44:05.598+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:44:05.598+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:44:05.713+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:44:05.712+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T10:44:05.714+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:44:05.713+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T10:44:05.714+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:44:07.173+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.592 seconds
[2023-08-05T10:47:18.926+0200] {processor.py:157} INFO - Started process (PID=82840) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:47:18.932+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T10:47:18.933+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:47:18.933+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:47:18.984+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:47:18.983+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T10:47:18.984+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:47:18.984+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T10:47:18.984+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:47:19.695+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.770 seconds
[2023-08-05T10:49:36.327+0200] {processor.py:157} INFO - Started process (PID=82902) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:49:36.333+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T10:49:36.334+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:49:36.334+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:49:36.421+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:49:36.420+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T10:49:36.421+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:49:36.421+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T10:49:36.421+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:49:37.431+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.105 seconds
[2023-08-05T10:54:02.327+0200] {processor.py:157} INFO - Started process (PID=83021) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:54:02.332+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T10:54:02.333+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:54:02.333+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:54:02.390+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:54:02.389+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T10:54:02.390+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:54:02.390+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T10:54:02.390+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:54:03.067+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.742 seconds
[2023-08-05T10:56:21.105+0200] {processor.py:157} INFO - Started process (PID=83086) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:56:21.112+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T10:56:21.113+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:56:21.113+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:56:21.209+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:56:21.201+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T10:56:21.209+0200] {logging_mixin.py:150} INFO - [2023-08-05T10:56:21.209+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T10:56:21.209+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T10:56:22.160+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.057 seconds
[2023-08-05T11:12:50.791+0200] {processor.py:157} INFO - Started process (PID=83280) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:12:50.807+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:12:50.810+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:12:50.809+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:12:50.894+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:12:50.894+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:12:50.894+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:12:50.894+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:12:50.895+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:12:51.755+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.965 seconds
[2023-08-05T11:17:08.267+0200] {processor.py:157} INFO - Started process (PID=83403) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:17:08.272+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:17:08.273+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:17:08.273+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:17:08.327+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:17:08.327+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:17:08.328+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:17:08.328+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:17:08.328+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:17:08.986+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.721 seconds
[2023-08-05T11:21:42.474+0200] {processor.py:157} INFO - Started process (PID=83538) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:21:42.481+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:21:42.482+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:21:42.482+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:21:42.539+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:21:42.539+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:21:42.540+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:21:42.540+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:21:42.540+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:21:43.507+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.034 seconds
[2023-08-05T11:24:01.024+0200] {processor.py:157} INFO - Started process (PID=83594) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:24:01.030+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:24:01.032+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:24:01.031+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:24:01.092+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:24:01.092+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:24:01.092+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:24:01.092+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:24:01.093+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:24:02.040+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.017 seconds
[2023-08-05T11:28:25.891+0200] {processor.py:157} INFO - Started process (PID=83709) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:28:25.897+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:28:25.898+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:28:25.897+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:28:25.953+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:28:25.952+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:28:25.953+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:28:25.953+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:28:25.953+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:28:26.692+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.802 seconds
[2023-08-05T11:33:51.529+0200] {processor.py:157} INFO - Started process (PID=83880) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:33:51.538+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:33:51.539+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:33:51.539+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:33:51.597+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:33:51.597+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:33:51.598+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:33:51.598+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:33:51.598+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:33:52.351+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.824 seconds
[2023-08-05T11:36:13.798+0200] {processor.py:157} INFO - Started process (PID=83960) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:36:13.809+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:36:13.810+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:36:13.810+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:36:13.887+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:36:13.886+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:36:13.887+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:36:13.887+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:36:13.887+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:36:14.785+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.988 seconds
[2023-08-05T11:38:21.756+0200] {processor.py:157} INFO - Started process (PID=84019) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:38:21.766+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:38:21.767+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:38:21.766+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:38:21.849+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:38:21.849+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:38:21.850+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:38:21.850+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:38:21.850+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:38:22.540+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.789 seconds
[2023-08-05T11:42:45.366+0200] {processor.py:157} INFO - Started process (PID=84153) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:42:45.372+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:42:45.373+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:42:45.373+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:42:45.430+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:42:45.429+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:42:45.430+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:42:45.430+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:42:45.430+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:42:46.093+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.728 seconds
[2023-08-05T11:44:53.198+0200] {processor.py:157} INFO - Started process (PID=84216) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:44:53.204+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:44:53.205+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:44:53.205+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:44:53.254+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:44:53.254+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:44:53.255+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:44:53.255+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:44:53.255+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:44:53.958+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.762 seconds
[2023-08-05T11:50:06.919+0200] {processor.py:157} INFO - Started process (PID=84397) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:50:06.925+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:50:06.926+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:50:06.925+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:50:06.979+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:50:06.978+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:50:06.979+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:50:06.979+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:50:06.979+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:50:07.675+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.758 seconds
[2023-08-05T11:52:22.187+0200] {processor.py:157} INFO - Started process (PID=84461) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:52:22.192+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:52:22.192+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:52:22.192+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:52:22.243+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:52:22.242+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:52:22.243+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:52:22.243+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:52:22.244+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:52:22.961+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.776 seconds
[2023-08-05T11:56:33.092+0200] {processor.py:157} INFO - Started process (PID=84584) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:56:33.101+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T11:56:33.102+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:56:33.102+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:56:33.154+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:56:33.153+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T11:56:33.154+0200] {logging_mixin.py:150} INFO - [2023-08-05T11:56:33.154+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T11:56:33.154+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T11:56:33.826+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.735 seconds
[2023-08-05T12:02:21.603+0200] {processor.py:157} INFO - Started process (PID=84808) to work on /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T12:02:21.610+0200] {processor.py:826} INFO - Processing file /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-05T12:02:21.611+0200] {logging_mixin.py:150} INFO - [2023-08-05T12:02:21.611+0200] {dagbag.py:541} INFO - Filling up the DagBag from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T12:02:21.677+0200] {logging_mixin.py:150} INFO - [2023-08-05T12:02:21.676+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-05T12:02:21.677+0200] {logging_mixin.py:150} INFO - [2023-08-05T12:02:21.677+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-05T12:02:21.677+0200] {processor.py:838} WARNING - No viable dags retrieved from /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-05T12:02:22.451+0200] {processor.py:179} INFO - Processing /Users/felipe.vasconcelos/repos/courses/python-learnings/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.849 seconds
